photo-points/Views/Admin/newView.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@model photo_points.Models.User
+@addTagHelper *,Microsfot.AspNetCore.Mvc.TagHelpers
```

Comment:
*Microsoft
Probably needs to be fixed on the other view pages, as well, but overall looks good.

Reply:
Thanks Eric!